apps/geo/serializers.py

```diff
@@ -148,3 +148,38 @@ class Meta:
             'region_title', 'admin_level_level', 'admin_level_title',
             'parent'
         )
+
+
+class RegionGqSerializer(ProjectPropertySerializerMixin, UserResourceSerializer):
+    project = serializers.IntegerField()
+
+    class Meta:
+        model = Region
+        fields = '__all__'
```

Comment:
Please look at the existing Serializer used by REST.

We also need to add `exclude = ('geo_options',)`